distribution/ddf-common/src/main/resources/bin/ddf.bat

```diff
@@ -3,42 +3,68 @@ SETLOCAL
 
 SET ARGS=%*
 SET DIRNAME=%~dp0%
-SET SOLR_PID="0"
+SET SOLR_PORT="8994"
+SET SOLR_MANAGED_INTERNALLY="true"
+SET SOLR_CLIENT=""
```

Comment:
Why set the global vars here when they are set later around line 23?

Reply:
Defaults if it can't find them ( or they get deleted from the system.properties file)

Reply:
That could be confusing. Imagine I'm an admin. I thought I set the port to 8383 in the system.properties file, but I accidentally added a letter to the property name. Now but the darn thing keeps starting on port 8994 and I can't figure out why.

Reply:
Line 44